1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,15 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for (let property in salaries){
```

Comment:
Try to think of a clear name for the variable rather than `property`. Think about what that variable represents!